pyproject.toml

```
@@ -25,21 +25,9 @@ requires = [
 
     # now matches minimum supported version, keep here as separate requirement
     # to be able to sync more easily with oldest-supported-numpy
-    # loongarch64 requires numpy>=1.22.0
-    "numpy==1.22.0; platform_machine=='loongarch64'",
-
-    # On Windows we need to avoid 1.21.6, 1.22.0 and 1.22.1 because they were
-    # built with vc142. 1.22.3 is the first version that has 32-bit Windows
-    # wheels *and* was built with vc141. So use that:
-    "numpy==1.22.3; python_version=='3.9' and platform_system=='Windows' and platform_python_implementation != 'PyPy'",
-    "numpy==1.22.3; python_version=='3.10' and platform_system=='Windows' and platform_python_implementation != 'PyPy'",
 
     # default numpy requirements
-    "numpy==1.21.6; python_version=='3.9' and (platform_system!='Windows' and platform_machine!='loongarch64') and platform_python_implementation != 'PyPy'",
-    # Note that 1.21.3 was the first version with a complete set of 3.10 wheels,
-    # however macOS was broken and it's safe C API/ABI-wise to build against 1.21.6
-    # (see oldest-supported-numpy issues gh-28 and gh-45)
-    "numpy==1.21.6; python_version=='3.10' and (platform_system!='Windows' and platform_machine!='loongarch64') and platform_python_implementation != 'PyPy'",
+    "numpy==1.22.4; python_version<='3.10' and platform_python_implementation != 'PyPy'",
```

```suggestion
    "numpy==1.22.4; python_version<='3.10',
```

I think we can remove the PyPy restriction here, as PyPy has been supported before 1.22.4, and PyPy 3.10 has just come out (I fully expect it to be compatible, as were the previous updates)

I know this is supposed to by synced with https://github.com/scipy/oldest-supported-numpy, I made a similar [comment](https://github.com/scipy/oldest-supported-numpy/pull/69) there. From the POV of scipy, I don't see why we have to _require_ bleeding-edge numpy for PyPy (even in the absence of published wheels), if older ones have a fully passing test suite on PyPy since at least 1.19

---

There are no `pp310` wheels for `numpy` yet, so I don't want to `==` require an old version. It also shouldn't matter anymore with numpy 1.25.0, that should be backwards and forwards-compatible C ABI wise, so that's an extra reason not to change it now.

Let's not change anything here until we change it in `oldest-supported-numpy` first.